tests/test_hubspot_automatic_fields_test.py

```diff
@@ -11,19 +11,12 @@ def name(self):
 
     def get_properties(self):
         return {
-            'start_date' : '2016-06-02T00:00:00Z',
+            'start_date' : '2021-05-02T00:00:00Z',
         }
 
     def expected_streams(self):
-        return self.expected_check_streams().difference({
-            'subscription_changes',
-            'email_events',
-            'companies',
-            'contacts',
-            'deals',
-            'engagements',
-            'contacts_by_company',
-        })
+        """streams to test"""
+        return self.expected_check_streams()
```

Comment:
If all streams are under test we should get rid of this method and call expected_check_streams directly in the test.